azext_iot/central/commands_scheduled_job.py

```diff
@@ -0,0 +1,185 @@
+from azext_iot.common import utility
+from azext_iot.central.common import API_VERSION
+from azext_iot.central.models.ga_2022_07_31 import ScheduledJobGa
+
+
+def get_scheduled_job(
+    cmd,
+    app_id: str,
+    job_id: str,
+    token=None,
+    central_dns_suffix=CENTRAL_ENDPOINT,
+    api_version=API_VERSION,
+) -> ScheduledJobGa:
+    provider = CentralScheduledJobProvider(
+        cmd=cmd, app_id=app_id, token=token, api_version=api_version
+    )
+
+    return provider.get_scheduled_job(
+        job_id=job_id,
+        central_dns_suffix=central_dns_suffix,
+    )
+
+
+def list_scheduled_jobs(
+    cmd,
+    app_id: str,
+    token=None,
+    central_dns_suffix=CENTRAL_ENDPOINT,
+    api_version=API_VERSION,
+) -> List[ScheduledJobGa]:
+    provider = CentralScheduledJobProvider(
+        cmd=cmd, app_id=app_id, token=token, api_version=api_version
+    )
+
+    return provider.list_scheduled_jobs(central_dns_suffix=central_dns_suffix)
+
+
+def create_scheduled_job(
+    cmd,
+    app_id: str,
+    job_id: str,
+    group_id: str,
+    schedule: str,
+    content: str,
+    job_name=None,
+    description=None,
+    batch_type=None,
+    threshold_type=None,
+    threshold_batch=None,
+    batch=None,
+    threshold=None,
+    token=None,
+    central_dns_suffix=CENTRAL_ENDPOINT,
+    api_version=API_VERSION,
+) -> ScheduledJobGa:
+    if not isinstance(content, str):
+        raise InvalidArgumentValueError("content must be a string: {}".format(content))
+
+    if not isinstance(schedule, str):
+        raise InvalidArgumentValueError("content must be a string: {}".format(content))
+
+    payload = utility.process_json_arg(content, argument_name="content")
+
+    schedule = utility.process_json_arg(schedule, argument_name="schedule")
+
+    provider = CentralScheduledJobProvider(
+        cmd=cmd, app_id=app_id, api_version=api_version, token=token
+    )
+
+    return provider.create_scheduled_job(
+        job_id=job_id,
+        job_name=job_name,
+        group_id=group_id,
+        content=payload,
+        schedule=schedule,
+        description=description,
+        batch_percentage=True
+        if batch_type is not None and batch_type.lower() == "percentage"
+        else False,
+        threshold_percentage=True
+        if threshold_type is not None and threshold_type.lower() == "percentage"
+        else False,
+        threshold_batch=threshold_batch,
+        batch=batch,
+        threshold=threshold,
+        central_dns_suffix=central_dns_suffix,
+    )
+
+
+def update_scheduled_job(
+    cmd,
+    app_id: str,
+    job_id: str,
+    group_id=None,
+    schedule=None,
+    content=None,
+    job_name=None,
+    description=None,
+    batch_type=None,
+    threshold_type=None,
+    threshold_batch=None,
+    batch=None,
+    threshold=None,
+    token=None,
+    central_dns_suffix=CENTRAL_ENDPOINT,
+    api_version=API_VERSION,
+) -> ScheduledJobGa:
+    payload = None
+    if content is not None:
```

Comment:
you could say `if content` to achieve the same thing. The only time I use `is not None` is if I want different behavior from something that resolves to false (like an empty string) vs None object

Reply:
Sounds good.